docs/topics/vscode-extension-configuring-analysis.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * docs/vs-code-extension-guide/master.adoc
+
+[id='vscode-extension-configuring-analysis_{context}']
+= Configuring a run configuration
+
+You can configure multiple run configurations to run against each project.
+
+.Prerequisites
+
+* Projects imported into VS Code.
+
+.Procedure
+
+. In *Extensions* view, click the *{ProductName}* icon (image:vs_mta_ex_icon.png[MTA code extension]) on the Activity bar.
+. Click the *+* (plus sign) next to *{ProductName}* to add a run configuration.
+. Complete the following configuration fields:
+
+* *Input*: Click *Add* and either:
```

Comment:
"and either" is not a style that we use. Suggest "do one of the following options" or actions.